frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -76,10 +76,14 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   }, [encryption.clusterWide, encryption.storageClass, encryptionChecked]);
 
   const toggleEncryption = (checked: boolean) => {
-    setEncryptionDispatch(ActionType.SET_ENCRYPTION, mode, dispatch, {
+    const encryptOj = {
       ...encryption,
       clusterWide: checked,
-    });
+    }
+    if(!checked){
+      encryptOj.advanced = false;
```

Comment:
why you want to set `advance encryption` to `true` when checked is set to `false`?

Reply:
I am setting advance encryption false only when Encryption is unchecked or set to false